monitoring/collector/systemd.go

```diff
@@ -0,0 +1,130 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+import (
+	"github.com/gravitational/satellite/monitoring"
+
+	"github.com/coreos/go-systemd/dbus"
+	"github.com/gravitational/trace"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var listOfServices = map[string]int{
+	"flanneld.service":                1,
```

Comment:
technically, as you're not using the value, you could go with `map[string]struct{}`:
```go
var listOfServices = serviceMap(
"flanneld.service",
"serf.service",
"planet-agent.service",
)

func serviceMap(services ...string) map[string]struct{} {
  result := map[string]struct{}
  for _, service := range services {
    result[service] = struct{}{}
  }
  return result
}
```